test/runtests.jl

```diff
@@ -3,15 +3,32 @@ using FixedSizeArrays
 using OffsetArrays: OffsetArray
 import Aqua
 
+# Check if the compilation options allow maximum performance.
+const build_is_production_build_env_name = "BUILD_IS_PRODUCTION_BUILD"
+const build_is_production_build = if haskey(ENV, build_is_production_build_env_name)
+    let v = ENV[build_is_production_build_env_name]
```

Comment:
```suggestion
const build_is_production_build = let v = get(ENV, build_is_production_build_env_name, false)
```
should probably reindent all the rest, and remove the `else; false` branch.